bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -53,7 +56,8 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     protected @Nullable ThingHandler createHandler(Thing thing) {
         ThingTypeUID thingTypeUID = thing.getThingTypeUID();
         if (SUPPORTED_THING_TYPES_UIDS.contains(thingTypeUID)) {
-            return new BigAssFanHandler(thing, networkAddressService.getPrimaryIpv4HostAddress());
+            String ipAddress = networkAddressService.getPrimaryIpv4HostAddress();
+            return new BigAssFanHandler(thing, ipAddress);
```

Comment:
What was the point in breaking this into 2 lines?

Reply:
Same cause as the logger. Don't think it is wrong, but you're right, old was better.